### ISSUE TITLE
Duplicating library for "web site"

### ISSUE THREAD

**Issue by bigbosst** [openaudible/openaudible#1306]
open audible is trying to create an entire second copy of my audio library to create a web page. It's copying all my ./books from a folder called books into ./web/m4b this doesn't make sense and is an obvious bug to try and duplicate my entire library. This is something new that was introduced and is happening in 3.9.6 after any audible library sync.

**Comment by murraypetera** [comment]
It would be really great that the web page would use the same files and not duplicate the books!

+1 on this feature

**Comment by oboe45** [comment]
Yes, I agree, and, as discussed lots of times in the past few years, a
better web product - drop down menus for selections, configurable columns,
include series and counts and of course, build in security on the generated
web page.

**Comment by murraypetera** [comment]
simple link "ls -s " to the files in the m4b folder to the web folder would do the trick for 99% of the dupe disk usage....

**Comment by Kagee** [comment]
or just ls / mklink (windows) the whole folder

**Comment by oboe45** [comment]
re /l /mkink - does this, or something similar work on a Mac??

**Comment by bigbosst** [comment]
On Linux, Unix, and MacOS you create symbolic links, symlinks, with `ln -s` you could run into security errors or missing the actual data trying to serve links to files as a web page. But it should be ready to test.

**Comment by openaudible** [comment]
Improved web export is a known feature request. Closing for now.
